raiden/network/transport/matrix/transport.py

```diff
@@ -821,6 +821,12 @@ def _handle_message(self, room: Room, event: Dict[str, Any]) -> bool:
             )
             return False
 
+        if self._is_broadcast_room(room):
+            # This must not happen. Nodes must not listen on broadcast rooms.
+            raise RuntimeError(
```

Comment:
Are the global rooms created as "read-only" rooms? can i create a custom client to send a message to those global rooms? If so, i could crash the whole network with this exception.

Apparently, [i can](https://github.com/raiden-network/raiden/pull/5343/files#diff-c548bdd4678475e829606a49d40f34d3R1086)

Reply:
You can't after this PR (and not currently as well, see my explanation in #5276)

Reply:
Makes sense. So we shouldn't be listening for messages on a broadcast room to begin with.